core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedDocServiceTest.java

```diff
@@ -354,7 +373,7 @@ private static void addMarkdownDescriptionMethodInfo(Map<Class<?>, Set<MethodInf
                          .descriptionInfo(DescriptionInfo.of("DESCRIPTION `PARAM`", Markup.MARKDOWN))
                          .build());
         final MethodInfo methodInfo = new MethodInfo(
-                "description", STRING, fieldInfos, ImmutableList.of(),
+                MyService.class.getName(), "description", 0, STRING, fieldInfos, ImmutableList.of(),
```

Comment:
just double-checking, does this include the package name?

Reply:
Yes and let me show later how we find the corresponding struct in `StructInfo` or `enum` in `EnumInfo` later. 😉